apps/web/playwright/login.2fa.e2e.ts

```diff
@@ -44,7 +44,10 @@ test.describe("2FA Tests", async () => {
         secret: secret!,
       });
 
-      await expect(page.locator(`[data-testid=two-factor-switch][data-state="checked"]`)).toBeVisible();
+      await page.waitForSelector(`[data-testid=two-factor-switch]`);
+      await expect(page.locator(`[data-testid=two-factor-switch]`)).toBeTruthy();
```

Comment:
Rather than just waiting for the switch to appear. Also expect that it is set to `true`

Reply:
I don't think this checks if it is checked- rather whether the locator for the switch is truthy, so whenever it's present. There's an `isChecked` param in page. though.